packages/side-nav/src/SideNavGroup.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { css, cx } from '@leafygreen-ui/emotion';
+import { uiColors } from '@leafygreen-ui/palette';
+import { ulStyleOverrides, LEFT_RIGHT_OFFSET } from './styles';
+
+const sideNavLabelStyle = css`
+  display: block;
```

Comment:
We should get this for free so long as we're using a div or header. This is fine to keep if we switch to a label tag I believe.

Reply:
H4 it is, so we'll get this and bold for free, but pay for it in margin top and bottom.